File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/EntriesByRaceResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class EntriesByRaceResponse : IResponse
    {
        private readonly IEnumerable<RaceEntry> raceEntries;

        public EntriesByRaceResponse(IEnumerable<RaceEntry> raceEntries)
        {
            this.raceEntries = raceEntries;
        }

        public IEnumerable<RaceEntry> RaceEntries { get { return raceEntries; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab09/networking/src/main/java/com/ubb/protocol/response/EntriesByRaceResponse.java
package com.ubb.protocol.response;

import com.ubb.model.RaceEntry;

import java.util.Collection;

public class EntriesByRaceResponse implements Response {
    private final Collection<RaceEntry> entries;

    public EntriesByRaceResponse(Collection<RaceEntry> entries) {
        this.entries = entries;
    }

    public Collection<RaceEntry> getEntries() {
        return entries;
    }
}
